packages/api/src/webhooks/cannon.ts

```diff
@@ -69,7 +134,14 @@ export default class WebhookCannon {
       return;
     }
     try {
-      await this.onTrigger(message);
```

Comment:
`onTrigger` function is not used anymore - should be removed, and code handling `recording.ready` should be moved to `handleEventsQueue`